include/llvm-dialects/Dialect/Utils.h

```diff
@@ -78,4 +78,7 @@ template <typename T> Printable<T> printable(const T &x) {
   return Printable<T>(x);
 }
 
+bool isMatchingFunc(llvm::StringRef refFn, bool isOverload,
+                    llvm::StringRef mnemonic);
```

Comment:
See the other comment -- I don't think this is needed, just an `isOperationDecl` is fine and fits better to what we already have.